### PR TITLE
change compression

### PR DESCRIPTION
**Pull request by ylil93**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def replace_relative_with_absolute(match):
 
 setup(
     name='pybadges',
-    version='1.0.1',
+    version='1.1.1',
```
Comment (brianquinlan) [inline setup.py]:
```suggestion
    version='1.0.2',
```

**Comment by liyanhui1228** [comment]
Can we use other python builtin compress/decompress modules here? `gzip` should work (I didn't try it though). And if we are going with the current approach, probably we will need to remove the `import lzma` part from the file.

**Comment by brianquinlan** [comment]
> Can we use other python builtin compress/decompress modules here? `gzip` should work (I didn't try it though). And if we are going with the current approach, probably we will need to remove the `import lzma` part from the file.

 I'd be fine with adding gzip support (I actually removed it at one point because I didn't want a tonne of conditional code) but @ylil93  would have to research whether it is available in Python 3 on Debian.

The lzma import is protected by an if statement that is only run if we try to load a .xz file, so that should be OK. But could you double check @ylil93 ?

**Comment by ylil93** [comment]
I can confirm that the lzma import statement is wrapped with an if clause

**Comment by brianquinlan** [comment]
Could you squash when you merge?

**Pull request merged**